src/components/ranking-chart/ranking-chart-component.jsx

```diff
@@ -1,63 +1,115 @@
-    totalMinusVeryHigh: 'Human modification',
-    noModification: 'No human modification'
+  [RANKING_GROUPS_SLUGS.humanModification]: {
+    [RANKING_INDICATORS.veryHigh]: 'Very high human modification',
+    [RANKING_INDICATORS.totalMinusVeryHigh]: 'Human modification',
+    [RANKING_INDICATORS.noModification]: 'No human modification',
+  },
+  [RANKING_GROUPS_SLUGS.protection]: {
+    [RANKING_INDICATORS.protected]: 'Current protection',
+    [RANKING_INDICATORS.protectionNeeded]: 'Additional protection needed',
+    [RANKING_INDICATORS.protectionNotNeeded]: 'Non-formal protection needed',
   },
-  protection: {
-    protected: 'Current protection',
-    protectionNeeded: ' Additional protection needed',
-    protectionNotNeeded: 'Non-formal protection needed'
-  }
 };
 
-const categories = Object.keys(legendText);
-const headers = categories.concat('spi');
+const GROUPS_SLUGS = { spi: 'spi', ...RANKING_GROUPS_SLUGS };
+const GROUPS = [GROUPS_SLUGS.spi, GROUPS_SLUGS.species, GROUPS_SLUGS.humanModification, GROUPS_SLUGS.protection];
+export const SORT_OPTIONS = [
+  { label: 'species protection index', slug: RANKING_INDICATORS.spi, group: GROUPS_SLUGS.spi },
+  { label: 'proportion of non endemic species', slug: RANKING_INDICATORS.nonEndemic, group: GROUPS_SLUGS.species },
+  { label: 'proportion of endemic species', slug: RANKING_INDICATORS.endemic, group: GROUPS_SLUGS.species },
+  {
+    label: 'proportion of very high human modification',
+    slug: RANKING_INDICATORS.veryHigh,
+    group: GROUPS_SLUGS.humanModification,
+  },
+  {
+    label: 'proportion of human modification',
+    slug: RANKING_INDICATORS.totalMinusVeryHigh,
+    group: GROUPS_SLUGS.humanModification,
+  },
+  {
+    label: 'proportion of non human modification',
+    slug: RANKING_INDICATORS.noModification,
+    group: GROUPS_SLUGS.humanModification,
+  },
+  {
+    label: 'proportion of protection',
+    slug: RANKING_INDICATORS.protected,
+    group: GROUPS_SLUGS.protection,
+  },
+  {
+    label: 'proportion of additional protection needed',
+    slug: RANKING_INDICATORS.protectionNeeded,
+    group: GROUPS_SLUGS.protection,
+  },
+  {
+    label: 'proportion of Non-formal protection needed',
+    slug: RANKING_INDICATORS.protectionNotNeeded,
+    group: GROUPS_SLUGS.protection,
+  },
+];
+
+const categories = Object.keys(LEGEND_TEXT);
 
 const RankingChart = ({
   data,
   className,
   countryISO,
-  handleInfoClick,
-  sortRankingCategory,
-  handleSortClick,
-  handleCountryClick
+  handleFilterSelection,
+  selectedFilterOption,
+  handleSearchChange,
+  handleCountryClick,
+  scrollPosition,
+  searchTerm,
 }) => {
   const [hasScrolled, changeHasScrolled] = useState(false);
   const tableRef = useRef();
+  useEffect(() => {
+    if (scrollPosition > -1 && tableRef.current) {
+      const foundElement = document.getElementById(`country-${scrollPosition}`)
+      if (foundElement) {
+        foundElement.scrollIntoView();
```

Comment:
I'm having some issues, in all the browsers I'm testing (firefox, chrome and vivaldi), when `scrollIntoView` is triggered. The whole container of the ranking view is scrolled and some gap appears at the bottom of the page. Maybe this is due to the positioning of the various `div`s.

![image](https://user-images.githubusercontent.com/6906348/110612887-e9a35000-8190-11eb-924b-969776b1a5cf.png)